package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reach/dialog": "^0.15.3",
```

Comment:
I didn't know about *Reach UI*, but I believe it's a good choice because I [read](https://reach.tech/): "Each component is tested with Safari + VoiceOver, Firefox + NVDA, and Edge + JAWS." and it's great for accessibility purposes.

Reply:
It's used for the accessibility reasons😄